automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -102,6 +105,26 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLike w
     outputFieldJson.asText()
   }
 
+  def parseWorkflowOptionsFromMetadata(metadata: String): String = {
+    val mapper = new ObjectMapper()
+    mapper.registerModule(DefaultScalaModule)
+
+    val metadataJson = mapper.readTree(metadata)
+    metadataJson.get("submittedFiles").get("options").asText()
+  }
+
+  def parseRuntimeAttributeKeyFromMetadata(metadata: List[JsonNode], attributeKey: String): List[String] = {
```

Comment:
Personal preference: I would probably make this function (and `parseWorkerAssignedExecEvents` below) operate on a single call metadata `JsonNode` and move the `map` up into the calling function. My reasoning is, it keeps the "transform one entry" logic separate from the "iterate over a list" logic.